components/support/test/src/main/java/mozilla/components/support/test/KArgumentCaptor.kt

```diff
@@ -32,11 +32,12 @@ class KArgumentCaptor<out T : Any?>(
 
     @Suppress("UNCHECKED_CAST")
     fun capture(): T {
-        return captor.capture() ?: createInstance(tClass) as T
+        return captor.capture() ?: castNull()
     }
 }
 
-fun <T : Any> createInstance(kClass: KClass<T>): T {
+@Deprecated("Try `mock()` or create instance manually")
```

Comment:
I think it's also safe to just remove this seeing as there aren't [any other usages](https://searchfox.org/mozilla-mobile/search?path=&q=createInstance).

Reply:
I agree. Just used deprecation to make transition smoother for external clients (if there are any). :)

Will remove it.

Reply:
@jonalmeida Done.